snap/snapcraft.yaml

```diff
@@ -0,0 +1,192 @@
+name: musescore
```

Comment:
I am not sure anymore what is the rule for uppercasing

Reply:
Still "MuseScore"